x-pack/plugins/stack_alerts/public/rule_types/es_query/expression/es_query_expression.tsx

```diff
@@ -197,7 +197,7 @@ export const EsQueryExpression: React.FC<
         <h5>
           <FormattedMessage
             id="xpack.stackAlerts.esQuery.ui.selectIndexPrompt"
-            defaultMessage="Select an index and time field"
+            defaultMessage="Select indices"
```

Comment:
should this also be `Select indices and a time field`?

Reply:
At the moment, both the index threshold rule and ES query rule (with query DSL) will show "Select indices" as the label and "Select indices and a time field" in the placeholder text. For example:

![image](https://github.com/elastic/kibana/assets/26471269/27dddb0a-01df-402e-a78e-2feec73d944c)

We could alternatively put the same fuller string in both the label and the placeholder. IMO it looked a little old when you went back to edit a rule and the label said "...and a time field" when all you saw was the list of indices, so that's largely why I went with the short label. WDYT?

Reply:
Gotcha....that makes sense to me!